modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/xml/SchemaConstants.java

```diff
@@ -40,10 +40,6 @@
   public static final QName MULE_ABSTRACT_POOLING_PROFILE_TYPE =
       new QName(MULE_NAMESPACE, "abstractPoolingProfileType", MULE_PREFIX);
   public static final QName MULE_POOLING_PROFILE_TYPE = new QName(MULE_NAMESPACE, "pooling-profile", MULE_PREFIX);
-  public static final QName MULE_ABSTRACT_THREADING_PROFILE_TYPE =
```

Comment:
Check with MG if there is anything else to be cleaned up in modules aside of mule-core related to threading profile.